client/src/lib/components/Buttons/IconButton/AnimatedIconButton.tsx

```diff
@@ -0,0 +1,88 @@
+import React, {useCallback, useEffect, useRef, useState} from 'react';
+import styled from 'styled-components/native';
+import AnimatedLottieView from 'lottie-react-native';
+import {SPACINGS} from '../../../constants/spacings';
+import BaseButton, {BaseButtonProps} from '../BaseButton';
+import {AnimatedIconType} from '../../Icons/AnimatedIcon';
+
+export type BaseIconButtonProps = BaseButtonProps & {
+  noBackground?: boolean;
+  fill?: string;
+};
+
+const StyledIconButton = styled(BaseButton)<BaseIconButtonProps>(props => ({
+  ...(props.noBackground ? {backgroundColor: 'transparent'} : {}),
+  width: props.small ? 36 : 44,
```

Comment:
Do we usually use the `SPACINGS` for height and width too? if so we could use them here I think

Reply:
Just coped it from the other buttons but true that